tests/test_number_parser_en.py

```
@@ -0,0 +1,25 @@
+import unittest
+from ovos_number_parser.numbers_en import numbers_to_digits_en, pronounce_number_en, pronounce_number_en
```

Ruff (0.8.2)

Redefinition of unused `pronounce_number_en` from line 2

Remove definition: `pronounce_number_en`

(F811)